src/GoogleMaps.xml

```diff
@@ -65,6 +65,11 @@
             <description>Returns the list of locations to display</description>
             <returnType type="Object" entityProperty="locationsEntity" isList="true"/>
         </property>
+        <property key="autoZoom" type="boolean" defaultValue="true">
```

Comment:
Add visibilityProps in src/GoogleMaps.webmodeler.ts to hide zoom configurations when autoZoom is true.